src/mumble/AudioConfigDialog.cpp

```diff
@@ -349,7 +349,7 @@ void AudioInputDialog::updateBitrate() {
 	overhead = 100 * 8 * (20 + 8 + 4 + 1 + 2 + p);
 
 	// TCP is 12 more bytes than UDP
-	if (NetworkConfig::TcpModeEnabled())
+	if (NetworkConfig::UdpModeDisabled())
```

Comment:
I think technically this if should only check whether TCP is currently used for _incoming audio - whether it is used for outgoing packets doesn't matter. So either we do something like `NetworkConfig::UdpModeDisabled || <UdpDisabledForIncomingTraffic>` or we only check the latter but ensure that if UDP is disabled in general, we also set the variable for incoming (and outgoing) UDP mode accordingly.

(Actually, re-check that this code is about processing _incoming_ packets and if it is actually processing outgoing ones, then adapt my statement above accordingly)